pkg/apis/baremetal/v1alpha1/baremetalmachineproviderspec_types.go

```diff
@@ -33,8 +33,12 @@ type BareMetalMachineProviderSpec struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// A custom deploy procedure.
+	// +optional
+	CustomDeploy *CustomDeploy `json:"customDeploy,omitempty"`
```

Comment:
Since this project is openshift-specific, maybe cut the corners and assume that the custom deploy method is always `install_coreos` and that we don't have an image?

Reply:
Speaking of image, the CBO-spawned metal3 pod caches the image in a different location than the installer. The location where the images are available is now looks like, for e.g.:

http://172.22.0.3:6181/images/ironic-python-agent.iso

How can we tell the agent about the new image location?

Reply:
BMO accepts the image URL in the `DEPLOY_ISO` variable.